pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.596.1</version>
```

Comment:
Firstly, that is a rather obnoxious comment IMO.  Relax a little regarding the rhetoric referencing people's abilities to explain things.  It comes off as condescending.  I sure hope that is not your intent.  It was a simple explanation.  I am attempting to solve a problem for functionality I need in a plugin. 

I was just explaining the version.  If you look at previous attempts at adding this functionality, of which failed due to various reason, at least one being a possible xstream incompatibility.  The version was bumped to pick up a newer version.  I believe this(these) PR are referenced in this PR.  If not, please look at https://github.com/jenkinsci/throttle-concurrent-builds-plugin/pull/33.  Remaining at 1.424 does not allow the current functionality to work (test breakage).  The code your are reviewing is a compilation of various others and is based on the logic in PR#33.  

Outside of this, the problem is solved, functionality is present, it just relies on moving jenkins up.

If you think you can solve the existing issues, using the existing version, feel free.  I have a version to use that works with this functionality and I am good with that.